pkgs/applications/misc/pwsafe/default.nix

```diff
@@ -0,0 +1,80 @@
+{ stdenv, fetchurl, wxGTK, libuuid, xercesc, zip,
+  libXt, libXtst, libXi, xextproto, gettext, perl,
```

Comment:
New lines should begin with a comma in order to align with the curly, remove the last comma above. You can fill each line to 72 chars.